filters-share/extended_css_ELEMHIDE.txt

```diff
@@ -1,4 +1,4 @@
-namu.wiki#?#article div:has(> div.wiki-paragraph) ~ div:has(table > tbody > tr > td[class] > div > img[src*="w.namu.la/"])
+namu.wiki#?#article div:has(a[href*="/w/%EB%B6%84%EB%A5%98:"]):has(> *:contains(분류)):not([class]) ~ div:has(table > tbody > tr > td[colspan]:not([style]) div:has(img[src*="ww.namu.la/s/"][class]):not(:has(div.namuwiki-toc-ad)))
```

Comment:
I suggest chaging to the following rule using two reasons:

1. namuwiki is continuously updating the DOM and this causes us to use filters performance-heavy.
2. to avoid the continuous update of the filter, instead of responding to the DOM changes, it's reasonable to change the general filter form to detect the `table` tag itself.

```css
##div:has(> table > tbody > tr > td span > img[src^="//ww.namu.la/s/"])
```

Reply:
Can you give a diff using suggestion feature of GitHub?